spec/presenters/two_factor_auth_code/phone_delivery_presenter_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+describe TwoFactorAuthCode::PhoneDeliveryPresenter do
+  let(:presenter) do
+    TwoFactorAuthCode::PhoneDeliveryPresenter.new(
+      code_value: '123',
+      delivery_method: 'sms',
+      phone_number: '123-123-1234'
+    )
+  end
+
+  it 'implements GenericDeliveryPresenter' do
+    TwoFactorAuthCode::PhoneDeliveryPresenter.instance_methods(false).each do |m|
+      expect(presenter).to respond_to m
+    end
+  end
+
+  it 'is a subclass of GenericDeliveryPresenter' do
+    expect(TwoFactorAuthCode::PhoneDeliveryPresenter.superclass).to(
+      be(TwoFactorAuthCode::GenericDeliveryPresenter)
+    )
+  end
+end
```

Comment:
These are tested in the views that use them, is it better to test them directly here in lieu of in the views, or in both?